### PR TITLE
OSDOCS#8789: Remove upgrade blocks for vSphere migration

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.15+

Issue: https://issues.redhat.com/browse/OSDOCS-8789

Link to docs preview: https://68558--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-migration

QE review:
- [ ] QE has approved this change.

Additional information:

**PTAL**:  @gcharot, @gnufied, @duanwei33

### CODE REVIEW

File: storage/container_storage_interface/persistent-storage-csi-migration.adoc [hunk 1]
```diff
@@ -11,10 +11,4 @@ In-tree storage drivers that are traditionally shipped with {product-title} are
...
 
-To change the default storage class, see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class].
-
-include::modules/persistent-storage-csi-migration-vsphere.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-* xref:../../updating/updating_a_cluster/eus-eus-update.adoc#eus-eus-update[Performing an EUS-to-EUS update]
\ No newline at end of file
+To change the default storage class, see xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class].
```
Comment (mburke5678) [inline storage/container_storage_interface/persistent-storage-csi-migration.adoc:14]: We're [not supposed to use intro text ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections)in additional references (even if it is helpful).
```suggestion
* xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class].
```

**Comment by mburke5678** [comment]
@lpettyjo One formatting issue. Otherwise LGTM

**Comment by gcharot** [comment]
LGTM

File: storage/container_storage_interface/persistent-storage-csi-migration.adoc (hunk 1 above)
Comment (lpettyjo) [reply storage/container_storage_interface/persistent-storage-csi-migration.adoc:14]: Got it. Thanks!

**Comment by duanwei33** [comment]
LGTM

File: storage/persistent_storage/persistent-storage-vsphere.adoc [hunk 2]
```diff
@@ -26,11 +26,11 @@ requested by users.
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree (non-CSI) plugin to provision vSphere storage.
+{product-title} 4.13 and later provides automatic migration for the vSphere in-tree volume plugin to its equivalent CSI driver.
```
[the remark this answers is not in the document]
Comment (lpettyjo) [reply storage/persistent_storage/persistent-storage-vsphere.adoc:29]: I've revised this content.

**Pull request merged**